.npmrc.template

```diff
@@ -0,0 +1,2 @@
+@fortawesome:registry=https://npm.fontawesome.com/
+//npm.fontawesome.com/:_authToken=${FONTAWESOME_PACKAGE_TOKEN}
```

Comment:
I think since `.npmrc` is part of the repo (and it pulls from the env var) we don't need this template after all